language-support/java/codegen/src/main/scala/com/digitalasset/daml/lf/codegen/CodeGenRunner.scala

```diff
@@ -109,6 +109,78 @@ object CodeGenRunner extends StrictLogging {
     logger.warn(s"Finish writing file '$outputFile'")
   }
 
+  /** Given the package prefixes specified per DAR and the module-prefixes specified in
+    * daml.yaml, produce the combined prefixes per package id.
+    */
+  def resolvePackagePrefixes(
```

Comment:
Would it make sense to restrict the visibility of this to either `private` or `private[codegen]` if you need to test this?